brave/brave-ios
# Fix #5106: Wallet edit site connection integration

## Summary of Changes
Integrated dapps edit site connection screen.

This PR fixes: #5106

## Submitter Checklist:


## Test Plan:
1. Connect Brave Wallet with a dapp site
2. Open wallet panel and click `Connect` or `Connected...` button
3. (It should bring up the edit site connection screen)
4. Check if dapp site origin is being displayed
5. Check if number of "connected" (it actually means "permitted") is correct 
6. Check if button action behave correctly 
7. Check if dismiss this screen with some changes of the account permission reflect back to the wallet panel

## Screenshots:
![simulator_screenshot_3D63B2F8-CAD6-4D35-B3BB-E05C727AC2E6](https://user-images.githubusercontent.com/1187676/161078575-8b287b06-4100-457e-bd75-8b2d9f01604d.png)

## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
